### ISSUE TITLE
Can't update from Webui

### ISSUE THREAD

**Issue by Keeze** [alf45tar/PedalinoMini#451]
Hi,

I try to update to the newest firmware via the web-interface. After pressin the button update and confirming I get the Progress=page, but nothing happens, even If i wait for a long time.

How can I solve this?

Cheers,
kees

**Comment by alf45tar** [comment]
Update firmware from web is not supported in all cases. It depends by board memory and how big is the firmware. The firmware size must be less than 50% of the memory amount.
Which firmware and which board are you using?

**Comment by alf45tar** [comment]
Only firmware compiled with

`board_build.partitions = src/partitions_ota.csv`

support web OTA update.

**Comment by Keeze** [comment]
[I use the firmware esp32doit-devkit on a ESP32DOIT Devkit ](https://github.com/alf45tar/PedalinoMini/tree/master/firmware/esp32doit-devkit-v1)

**Comment by alf45tar** [comment]
Did you try to reset the EEPROM to factory default?

**Comment by alf45tar** [comment]
Before any update save configuration, reset to factory default and reload configuration. EEPROM can change without any further advice.

**Comment by Keeze** [comment]
Nope, I am not, but looking at the article, on 3.3V I need 1.8K resistors?

**Comment by Keeze** [comment]
Resistors have been bodged and testing have been done ;) It seems I have a working pedalboard again. Tnx for your help!
